eslint.config.mjs

```diff
@@ -0,0 +1,124 @@
+import globals from 'globals';
+
+import js from '@eslint/js';
+import babelParser from '@babel/eslint-parser';
+
+import jsdoc from 'eslint-plugin-jsdoc';
+import html from 'eslint-plugin-html';
+import ava from 'eslint-plugin-ava';
+
+export default [
+  // Replaces .eslintignore
+  {
+    ignores: [
+      '**/*.min.js',
+      'common/',
+      'content/patterns/landmarks/examples/js/visua11y.js', // cspell:ignore visua
+      'content/shared/js/highlight.pack.js',
+      'content/shared/js/skipto.js',
+    ],
+  },
+  js.configs.recommended,
+  {
+    files: ['**/*.js'],
+
+    languageOptions: {
+      ecmaVersion: 2021,
+      parserOptions: {
+        ecmaFeatures: {
+          impliedStrict: false,
+        },
+      },
+      globals: {
+        ...globals.browser,
+      },
+      sourceType: 'script',
+    },
+
+    linterOptions: {
+      reportUnusedDisableDirectives: true,
```

Comment:
Or ['warn'](https://eslint.org/docs/latest/use/configure/migration-guide#linter-options) if wanting to be explicit. According to [Reporting Unused Disable Directives](https://eslint.org/docs/latest/use/configure/configuration-files#reporting-unused-disable-directives):

> For legacy compatibility, true is equivalent to "warn" ...

Reply:
```suggestion
      reportUnusedDisableDirectives: 'error',
```